src/components/nodes/pages/stanford-page/stanford-page-page.tsx

```diff
@@ -38,7 +38,9 @@ const StanfordPagePage = ({node, ...props}: Props) => {
   const lastComponent = node.suPageComponents && node.suPageComponents[node.suPageComponents.length - 1]
 
   const hasBannerOrCalculator =
-    lastComponent?.__typename?.includes("Banner") || lastComponent?.__typename?.includes("Calculator")
+    lastComponent?.__typename?.includes("Banner") ||
+    lastComponent?.__typename?.includes("Calculator") ||
+    lastComponent?.__typename?.includes("Testimonial")
```

Comment:
You could do `["ParagraphSumCalculator", "ParagraphSumTestimonial", "ParagraphStanfordBanner"].includes(lastComponent?.__typename)`